assets/dev/js/frontend/handlers/background-slideshow.js

```diff
@@ -64,7 +71,8 @@ export default class BackgroundSlideshow extends elementorModules.frontend.handl
 			direction = 'slide_left' === elementSettings.background_slideshow_slide_transition ? 'ltr' : 'rtl',
 			$container = jQuery( '<div>', { class: classes.swiperContainer, dir: direction } ),
 			$wrapper = jQuery( '<div>', { class: classes.swiperWrapper } ),
-			kenBurnsActive = elementSettings.background_slideshow_ken_burns;
+			kenBurnsActive = elementSettings.background_slideshow_ken_burns,
+			lazyload = 'yes' === elementSettings.background_slideshow_lazyload;
```

Comment:
Maybe we should create a method instead of repeating this code?
```
isLazyLoad(){
  return 'yes' === elementSettings.background_slideshow_lazyload;
}
```

Reply:
Can't use elementSettings outside the functions.